File: RetaguardaESilva/BackEnd/src/RetaguardaESilva.Domain/Models/Pedido.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography;
using System.Text;
using System.Threading.Tasks;

namespace RetaguardaESilva.Domain.Models
{
    public class Pedido
    {
        public int Id { get; set; }
        public int ClienteId { get; set; }
        public int ProdutoId { get; set; }
        public int FornecedorId { get; set; }
        public int TransportadorId { get; set; }
        public int EmpresaId { get; set; }
        public int UsuarioId { get; set; }
        public int Quantidade { get; set; }
        public decimal Valor { get; set; }
        public DateTime? DataCadastroPedido { get; set; }
        public bool Ativo { get; set; }
    }
}

File: RetaguardaESilva/FrontEnd/RetagurdaESilva-App/src/app/models/pedido.ts
import { Produto } from "./produto";
export interface Pedido {
  id:number;
  clienteNome:string;
  clienteId:number;
  transportadorNome:string;
  transportadorId:number;
  usuarioId:number;
  empresaId:number;
  precoTotal:number;
  dataCadastroPedido:string;
  statusPedido:string;
  produtos: Array<Produto>;
}
